wrapanapi/lenovo.py

```diff
@@ -358,5 +360,84 @@ def inventory(self, *requested_items, **kwargs):
         requested_items = requested_items or self._inventory_available
         return {item: self._inventory_available[item](self, requester) for item in requested_items}
 
+    def _get_network_devices(self, server_name):
```

Comment:
I don't think these need to be, or should be, `_method`, I think they can be 'public' like the other methods used for stat collection.

1. Please add docblocks with description of assumptions you're making about device (network and storage) lookup.  description of expected device attributes how to tell addin cards from pci cards in the resulting device list, things like that.

2. Make these functions 'public' (remove `_` prefix) unless there is some circumstance that prevents this.